### PR TITLE
(SIMP-2471) Add SCAP results to documentation

### PR DESCRIPTION
**Pull request by ralph-wright**
- Added client and server results to the security conop

SIMP-2471 #close

### CODE REVIEW

File: docs/security_conop/scans/SSG_STIG_SIMP_Client.rst [hunk 1]
```diff
@@ -0,0 +1,1772 @@
...
+Removing all ability for Root to login from serial ports prevents "last
+effort" recovery of remote systems. This is not something that SIMP will
+enable by default.
+
+You can make this compliant by setting the following in Hiera:
+
+.. code:: yaml
+
+    ---
+    simplib::securetty :
```
Comment (jeefberkey) [inline docs/security_conop/scans/SSG_STIG_SIMP_Client.rst]: This looks like it was rain against the SIMP 6 Alpha. 
@trevor-vaughan

Comment (trevor-vaughan) [inline docs/security_conop/scans/SSG_STIG_SIMP_Client.rst]: @jeefberkey Yes, it was. When we have a full Beta, we'll run it against that again

Comment (ralph-wright) [inline docs/security_conop/scans/SSG_STIG_SIMP_Client.rst]: @trevor-vaughan is it better to just wait for the Beta before doing anything with this?

Comment (trevor-vaughan) [inline docs/security_conop/scans/SSG_STIG_SIMP_Client.rst]: @ralph-wright Yeah, I think so. I'm going to go ahead and close this but at least we'll have the history.

**Pull request closed without merging**